### PR TITLE
PS-7840 Move the description of Adaptive Network Buffers into dedicat…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ed section (8.0)

modified:   source/index.rst
	new file:   source/performance/adaptive_network_buffers.rst
	modified:   source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst

### CODE REVIEW

File: doc/source/performance/adaptive_network_buffers.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Network buffers grow towards the `max_allowed_packet <https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_allowed_packet>`_ size and do not shrink until the connection is terminated. For example, if the connections are selected at random from the pool, an occasional big query eventually increases the buffers of all connections. The combination of `max_allowed packet` set to a value between 64MB to 128MB and the connection number between 256 to 1024 can create a large memory overhead.
+
+Percona Server| version 8.0.23-14 introduces the :variable:`net_buffer_shrink_interval` variable to solve this issue. The default value is 0 (zero). If you set the value higher than 0, Percona Server records the network buffer's maximum use size for the number of seconds set by `net_buffer_shrink_interval`. When the next interval starts, the network buffer is set to the recorded size. This action removes spikes in the buffer size.
```
Comment (nastena1606) [inline doc/source/performance/adaptive_network_buffers.rst]: |Percona Server|

**Review by nastena1606 — APPROVED** [review]
LGTM

File: doc/source/performance/adaptive_network_buffers.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/performance/adaptive_network_buffers.rst]: Fixed.

**Review by LenzGr — APPROVED** [review]
LGTM! Thank you for taking care of this.

**Pull request merged**